nodejs-server/src/operationsServer.ts

```diff
@@ -0,0 +1,172 @@
+    this.log(requestId, 'getOperation request:', request);
+    const operation = this.operations[request.name];
+    if (!operation) {
+      const error = new Error(`Operation ${request.name} does not exist`) as
+          grpc.ServiceError;
+      error.code = grpc.status.NOT_FOUND;
+      this.log(requestId, 'getOperation error:', error);
+      callback(error);
+      return;
+    }
+    const response = operation.getOperation();
+    this.log(requestId, 'getOperation response:', response);
+    callback(null, response);
+  }
+
+  deleteOperation(
+      call: grpc.ServerUnaryCall<longrunning.DeleteOperationRequest>,
+      callback: grpc.requestCallback<google.protobuf.Empty>): void {
+    const requestId = ++this.requestCount;
+    const request = call.request;
+    this.log(requestId, 'deleteOperation request:', request);
+    const operation = this.operations[request.name];
+    if (!operation) {
+      const error = new Error(`Operation ${request.name} does not exist`) as
+          grpc.ServiceError;
+      error.code = grpc.status.NOT_FOUND;
+      this.log(requestId, 'deleteOperation error:', error);
+      callback(error);
+      return;
+    }
+    delete this.operations[request.name];
+    const response = new google.protobuf.Empty();
+    this.log(requestId, 'deleteOperation response:', response);
+    callback(null, response);
+  }
+
+  cancelOperation(
+      call: grpc.ServerUnaryCall<longrunning.CancelOperationRequest>,
+      callback: grpc.requestCallback<google.protobuf.Empty>): void {
+    const requestId = ++this.requestCount;
+    const request = call.request;
+    this.log(requestId, 'cancelOperation request:', request);
+    const error =
+        new Error('CancelOperation is not implemented') as grpc.ServiceError;
+    error.code = grpc.status.UNIMPLEMENTED;
+    this.log(requestId, 'cancelOperation error:', error);
+    callback(error);
+  }
+
+  waitOperation(
+      call: grpc.ServerUnaryCall<longrunning.WaitOperationRequest>,
+      callback: grpc.requestCallback<longrunning.Operation>): void {
+    const requestId = ++this.requestCount;
+    const request = call.request;
+    this.log(requestId, 'waitOperation request:', request);
+    const operation = this.operations[request.name];
+    if (!operation) {
+      const error = new Error(`Operation ${request.name} does not exist`) as
+          grpc.ServiceError;
+      error.code = grpc.status.NOT_FOUND;
+      this.log(requestId, 'waitOperation error:', error);
+      callback(error);
+      return;
+    }
+    const response = operation.getOperation();
+    if (response.done) {
+      this.log(requestId, 'getOperation response:', response);
+      callback(null, response);
+      return;
+    }
+    this.log(requestId, 'getOperation waiting...');
+    operation.setCallback(() => {
+      const response = operation.getOperation();
+      this.log(requestId, 'getOperation response:', response);
+      callback(null, response);
+    });
+  }
+}
```

Comment:
newwwwwline

Reply:
we should totes make our linter catch this, and we both would have saved typing in messages :stuck_out_tongue_closed_eyes:

Reply:
The new `gts` (tagged `1.0.0-0`) checks and fixes it.